### PR TITLE
Remove references to the flag-for-reply workflow.

### PR DESCRIPTION
**Pull request by rmol**
## Status

Ready for review

## Description of Changes

Updates the docs after removal of the flag-for-reply workflow in https://github.com/freedomofpress/securedrop/pull/5954.

## Testing

Not much to check with deletions, but I am interested in whether others think we should retain the API endpoint documentation. I decided it was just confusing, since the endpoint now does nothing.

## Release 

This should be merged after https://github.com/freedomofpress/securedrop/pull/5954.

## Checklist (Optional)

- [x] Doc linting (`make docs-lint`) passed locally
- [x] Doc link linting (`make docs-linkcheck`) passed
- [x] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

**Comment by eloquence** [comment]
LGTM in principle once the main PR lands.

> I am interested in whether others think we should retain the API endpoint documentation. I decided it was just confusing, since the endpoint now does nothing.

As a matter of good practice, it may be worth noting in its own section, such as "Deprecated functionality", explaining that the no-op endpoint and the `is_flagged` property will be removed in a future release. Right now, the presence of the `is_flagged` property in the documentation is a bit confusing, given that it is always `false`.

**Review by rocodes — COMMENTED** [review]
LGTM. The only thing I would maybe suggest is that there are a couple sample JSON responses that include `is_flagged: false` and it could be good to add a comment (even inline in the code block) explaining that this is a legacy field that can be ignored.  

e.g. https://github.com/freedomofpress/securedrop-docs/pull/225/files#diff-466879439de016d374b3dc3cc22f5e587900a70cfdd5c80a1e51e35227510419L197-R197

**Comment by eloquence** [comment]
I've added a section to the end for now - @rocodes ready for another look from you

**Review by rocodes — APPROVED** [review]
WFM. Thank you @eloquence

**Pull request merged**
